odoo/cli/cloc.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import argparse
+import os
+import sys
+import textwrap
+
+from odoo.tools import cloc, config
+from . import Command
+
+class Cloc(Command):
+    """\
+    Odoo cloc is a tool to count the number of relevant lines written in
+    python, javascript or xml. This can be used as rough metric for pricing
```

Comment:
Python should be capitalized in the other occurrences as well, to be pedantic :P

```suggestion
    Python, Javascript or XML. This can be used as rough metric for pricing
```